tests/sentry/tasks/test_low_priority_symbolication.py

```diff
@@ -1,24 +1,164 @@
+import datetime
+from typing import TYPE_CHECKING, Any, Dict
+
 import pytest
+from freezegun import freeze_time
 
 from sentry.processing import realtime_metrics
-from sentry.tasks.low_priority_symbolication import _scan_for_suspect_projects, calculation_magic
+from sentry.processing.realtime_metrics.base import BucketedCount, DurationHistogram
+from sentry.processing.realtime_metrics.redis import RedisRealtimeMetricsStore
+from sentry.tasks.low_priority_symbolication import (
+    _scan_for_suspect_projects,
+    _update_lpq_eligibility,
+    calculation_magic,
+)
+from sentry.testutils import TestCase
 from sentry.testutils.helpers.task_runner import TaskRunner
-from sentry.utils import redis
 from sentry.utils.compat import mock
 
+if TYPE_CHECKING:
+    from typing import Callable
+
+    def _fixture(func: Callable[..., Any]) -> Callable[..., None]:
+        ...
+
+    pytest.fixture = _fixture
+
+
+def create_store() -> RedisRealtimeMetricsStore:
+    return RedisRealtimeMetricsStore(
+        cluster="default",
+        counter_bucket_size=10,
+        counter_ttl=datetime.timedelta(milliseconds=400),
+        histogram_bucket_size=10,
+        histogram_ttl=datetime.timedelta(milliseconds=400),
+    )
+
+
+STORE = create_store()
+
+
+def reset_store() -> None:
+    STORE = create_store()
+
+
+class TestScanForSuspectProjects(TestCase):
+    def tearDown(self):
+        reset_store()
+
+    @mock.patch("sentry.processing.realtime_metrics", STORE)
```

Comment:
unsure if this is excessive - i'm unsure if the store is reset in between every unit test so this is my attempt at guaranteeing this.